vagrant/nginx/app.conf

```diff
@@ -0,0 +1,75 @@
+server {
+   charset utf-8;
+   client_max_body_size 128M;
+
+   listen 80; ## listen for ipv4
+   #listen [::]:80 default_server ipv6only=on; ## listen for ipv6
+
+   server_name yii2-app-advanced-frontend.dev;
+   root        /app/frontend/web/;
+   index       index.php;
+
+   access_log  /app/vagrant/nginx/log/frontend-access.log;
+   error_log   /app/vagrant/nginx/log/frontend-error.log;
+
+   location / {
+       # Redirect everything that isn't a real file to index.php
+       try_files $uri $uri/ /index.php?$args;
+   }
+
+   # uncomment to avoid processing of calls to non-existing static files by Yii
+   #location ~ \.(js|css|png|jpg|gif|swf|ico|pdf|mov|fla|zip|rar)$ {
+   #    try_files $uri =404;
+   #}
+   #error_page 404 /404.html;
+
+   location ~ \.php$ {
+       include fastcgi_params;
+       fastcgi_param SCRIPT_FILENAME $document_root/$fastcgi_script_name;
```

Comment:
I use standard Yii config for nginx. I don't think we must customize it here. Each user can customize it (after git clone) for own requirements...